frontend/src/Home.tsx

```diff
@@ -4,8 +4,9 @@ import { ErrorComponent, Route } from "@tanstack/react-router";
 import axios, { AxiosError } from "axios";
 import { z } from "zod";
 import { userWithTimetablesType } from "../../lib/src/index";
+import { authenticatedRoute } from "./AuthenticatedRoute";
```

Comment:
```suggestion
import authenticatedRoute from "./AuthenticatedRoute";
```
Based on the above suggestion to fix the import.